index.js

```diff
@@ -10,4 +10,8 @@ module.exports = {
 
     forceHighlander(this.project);
   },
+
+  treeForAddon(tree) {
+    return this.processedAddonJsFiles(tree);
+  },
```

Comment:
Can you add a comment RE: not calling super (and not supporting templates or styles) and explaining what we are actually doing here?

Reply:
👍